docs/DeveloperGuide.md

```diff
@@ -976,6 +976,33 @@ The inverse of adding to favourites. The "star" in the top-right corner should c
        Expected: The application corresponding to `APP_INDEX` for the company at `COMPANY_INDEX` is removed.
        The application status of the company is changed to `CLOSED`. A message containing the details of the removed application is displayed in the results box.
 
+### Viewing full application details of a company
+
+1. View the full application details of a company
+
+   1. Requisites: The company corresponding to `INDEX` has at least one application.
+
+   1. Test case: `view INDEX` <br>
+      Expected: Only the selected company is shown in the company list. The `DESCRIPTIONS` of the applications of the 
```

Comment:
Suggestion to maintain DESCRIPTION instead of DESCRIPTIONS

Reply:
Good catch!